backend/account/views.py

```diff
@@ -1,3 +1,28 @@
+from django.contrib.auth.models import User
 from django.shortcuts import render
+from .serializers import UserRegisterSerializer
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
 
-# Create your views here.
+class Register(generics.CreateAPIView):
```

Comment:
Try to use something like RegisterView rather than Just Register to make it more clearer.
Also try to keep serializer name and view name kinda same like;
`RegisterView`
`RegisterSerializer`